src/frontend/http.rs

```diff
@@ -174,10 +186,11 @@ async fn key(Path(pk): Path<String>) -> Result<impl IntoResponse, AppError> {
 pub async fn start() -> Result<()> {
     let app = Router::new()
         .route("/remove/:pk/:blake3_hash", delete(remove_file))
-        .route("/store/:pk", post(post_file))
-        .route("/store_named/:pk/:name", post(post_file_named))
-        .route("/retrieve/:pk/:blake3_hash", get(get_file))
-        .route("/retrieve_named/:pk/:name", get(get_file_named))
+        .route("/hashed/:pk", post(post_file_hashed))
+        .route("/pubkeyed/:pk", post(post_file_pubkeyed))
+        .route("/named/:pk/:name", post(post_file_named))
+        .route("/hashed/:pk/:blake3_hash", get(get_file))
```

Comment:
Here it seems "get(get_file))" retrieves both "hashed" and "pubkeyed"

Reply:
Correct, they're both hashes